gui/src/ScenarioSelector.tsx

```diff
@@ -159,31 +217,89 @@ const DialogContentSx = {
   width: "500px",
 };
 
+const CancelBtnSx = {
```

Comment:
This can be defined outside the component

Reply:
This is already out of component definition. Do you want to move this as external file? @FredLL-Avaiga

Reply:
no it's fine, I didn't see the whole file when reviewing and I was expecting them at the top of the file :-)